### ISSUE TITLE
0.11.2 container crashlooping

### ISSUE THREAD

**Issue by daniel-ciaglia** [TierMobility/boring-registry#138]
Using Helm chart `version: 0.7.0`

Error as shown below

```
/boring-registry: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /boring-registry)
/boring-registry: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /boring-registry)
Stream closed EOF for boring-registry/boring-registry-596774b694-k9qkq (boring-registry)```

I didn't investigate much to be honest :(

**Comment by nc-ruth** [comment]
I am also experiencing the same issue when running the Docker container.
When I build manually (without Docker) it works fine.

**Comment by oliviermichaelis** [comment]
Hey @daniel-ciaglia ! Thanks for reporting the issue :)
I fixed this in [`v0.11.4`](https://github.com/TierMobility/boring-registry/pkgs/container/boring-registry/123528246?tag=v0.11.4) by compiling the binary without dynamically linking to glibc.

Let me know if that doesn't work for you!

**Comment by daniel-ciaglia** [comment]
Thanks a bunch, issue solved!
